### PR TITLE
Fix part of #5344: Enable Multiple Classrooms

### PR DESCRIPTION
**Pull request by theMr17**
## Explanation

Fixes part of #5344

Toggles the Multiple Classrooms Feature Flag on.

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by theMr17** [comment]
@adhiamboperes, I've created the PR but let's wait until the feature testing is completed before merging it.

**Comment by theMr17** [comment]
@adhiamboperes Not sure why the `Publish Code Coverage Report` job is failing.

/cc @Rd4dev.

**Comment by Rd4dev** [comment]
> @adhiamboperes Not sure why the `Publish Code Coverage Report` job is failing.
> 
> /cc @Rd4dev.

@theMr17, apologies for the setback. The coverage report publication encountered GitHub permission issues with PRs created from forks, although **it works fine with upstreams**. The issue has been identified, and I'm currently working on a fix. This should be resolved with my upcoming PR merge.

**Comment by adhiamboperes** [comment]
Please reasign once all checks pass.

**Comment by Rd4dev** [comment]
> Please reasign once all checks pass.

@adhiamboperes, apologies for the inconvenience. Just wanted to inform you about the code coverage report publication failure. This issue was recently identified with PRs pushed from forks due to GitHub permission constraints. I’m working on a fix, and it’s unrelated to the changes in this PR. (Though code coverage report publication will pass from upstream PRs)

**Comment by adhiamboperes** [comment]
Thanks RD.

This PR isn't due to be merged for a few days, so it is okay to wait for
your fix to be merged.

**Comment by Rd4dev** [comment]
> Thanks RD. This PR isn't due to be merged for a few days, so it is okay to wait for your fix to be merged.

Thank you so much for understanding. I’ll try to fix this ASAP.

**Comment by theMr17** [comment]
> Please reasign once all checks pass.

Seems like the check failure could be ignored for now, @adhiamboperes PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt [hunk 1]
```diff
@@ -544,6 +546,7 @@ class PinPasswordActivityTest {
 
   @Test
   fun testPinPassword_withUser_forgot_inputAdminPinAndNewPin_opensHomeActivity() {
+    TestPlatformParameterModule.forceEnableMultipleClassrooms(false)
```
Comment (adhiamboperes) [inline app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt:545]: While tehese tests are okay, is there any reason why the  corresponding `opensClassroomListActivity` tests are restricted to run on Espresso?

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @theMr17! PTAL at my comment.

File: app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt (hunk 1 above)
Comment (theMr17) [reply app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt:545]: Not sure, why those were added. I have removed the restrictions.

**Comment by theMr17** [comment]
@adhiamboperes PTAL.

**Review by adhiamboperes — APPROVED** [review]
LGTM, thanks.

**Comment by theMr17** [comment]
Closing this PR.

Re-created the PR from an upstream branch to prevent the `Publish Code Coverage Report` job from failing - #5510.

**Pull request closed without merging**